packages/util/src/staking/index.ts

```diff
@@ -97,17 +102,32 @@ export class StakingModule<T> {
     );
   }
 
+  /**
+   * Get history depth
```

Comment:
Could you add some comments on what is it and why should it be used?
Otherwise, this comment isn't needed